src/widgets/gauge.rs

```diff
@@ -104,3 +118,21 @@ impl<'a> Widget for Gauge<'a> {
         }
     }
 }
+
+#[test]
```

Comment:
Shouldn't those be put inside a `tests` module (as described here: https://doc.rust-lang.org/stable/book/2018-edition/ch11-03-test-organization.html) ? I thought that it was required for them not to be included in a normal build.